src/pages/HomePage/pages/MarketPlace/components/forms/ProductForm.tsx

```diff
@@ -48,155 +32,63 @@ export function ProductForm({
-                          value="+ Add another one"
-                          onClick={() =>
-                            push({
-                              color: "#000000",
-                              image: "",
-                            })
-                          }
-                          className="hover:no-underline"
-                        />
-                      </div>
-                    </div>
-                  )}
-                </FieldArray>
+              <Field
+                name="type"
+                component={FormikSelectField}
+                options={productTypes}
+                label="Product type"
+                id="type"
+                placeholder="Select Type"
+              />
 
-                <FormLayout>
-                  <Field
-                    name="price"
-                    component={FormikInputDiv}
-                    label="Price"
-                    id="price"
-                    placeholder=""
-                    type="number"
-                  />
-                  <Field
-                    name="stock"
-                    component={FormikInputDiv}
-                    label="Stock"
-                    id="stock"
-                    placeholder="Enter number of stocks"
-                    type="number"
-                  />
-                  <div className="space-y-2  ">
-                    <p className="font-semibold">Sizes</p>
-                    <ul className="flex gap-2 font-semibold">
-                      {sizes.map((size) => (
-                        <li
-                          key={size}
-                          className={`border text-center flex items-center justify-center size-9 rounded-lg  p-1 cursor-pointer ${
-                            values.sizes.includes(size)
-                              ? "bg-gray-200"
-                              : "bg-white"
-                          }`}
-                          onClick={() =>
-                            toggleSize(size, selectedSizes, setFieldValue)
-                          }
-                        >
-                          {size}
-                        </li>
-                      ))}
-                    </ul>
-                    {touched.sizes && errors.sizes && (
-                      <p className="text-error text-sma">{errors.sizes}</p>
-                    )}
-                  </div>
-                </FormLayout>
-              </div>
+              <Field
+                name="category"
+                component={FormikSelectField}
+                options={categories}
+                label="Product category"
+                id="category"
+                placeholder="Select Category"
+              />
+            </FormLayout>
+
+            <ProductGallery
+              values={values}
+              errors={errors}
+              touched={touched}
+              setFieldValue={setFieldValue}
+            />
+
+            <FormLayout>
```

Comment:
there should be only one FormLayout per form 
if there is a problem with the component please let me know